### DISCUSSION TITLE
Actual Database ID [Dec 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4733] [Discord Threads]
> 🙋‍♂️ **nhnkl3**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### enable `show system fields` from `fields` menu in toolbar  
#  
> 👨‍🦲 **dstala**  
####   
![image0](https://cdn.discordapp.com/attachments/1045334581743730720/1045342114730475631/Screenshot_2022-11-24_at_7.45.10_PM.png)  
#  
> 🙋‍♂️ **nhnkl3**  
#### oh god  
#  
> 🙋‍♂️ **nhnkl3**  
#### ty!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1045334581743730720)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
